vbezbakh/docker/vbezbakh:devopscampsummer2019_vbezbakh_docker_example6/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:latest
+
+RUN apt-get update
+RUN apt-get install -y python python-pip wget
+RUN pip install Flask
```

Comment:
Flask module isn't using here anywhere. Sure, it can be used as just as an CI image that contains required tools.